src/main/daml/Daml/Finance/Interface/Holding/Transferable.daml

```diff
@@ -44,14 +44,16 @@ interface Transferable where
     do
       pure $ view this
 
-  choice Transfer : ContractId Transferable
+  nonconsuming choice Transfer : ContractId Transferable
     -- ^ Transfer a contract to a new owner
     with
       newOwnerAccount : AccountKey
-        -- ^ Account contract id of the parties to transfer the contract to
-    controller getOwner this, getLocker this, newOwnerAccount.owner
+        -- ^ Account contract id of the parties to transfer the contract to.
+      transferors : Parties
```

Comment:
This name now overlaps with `transferors` in the data type from `Account`. And it doesn't _have to be_ the parties instructing and approving the transfer (as is written in the comment), eg it could be just the custodian. Maybe we call this just `actors`, and the comment reads "Parties authorizing the transfer.

Reply:
agree, good point